spot_wrapper/wrapper.py

```diff
@@ -1541,6 +1541,49 @@ def list_graph(self, upload_path):
             )
         ]
 
+    def clear_graph(self) -> typing.Tuple[bool, str]:
+        """Clear a graph in a robot.
+
+        Returns:
+            success (bool): success flag
+            message (str): message about result
+        """
+        try:
+            self._clear_graph()
+            return True, "Success"
+        except Exception as e:
+            return False, f"Error: {e}"
+
+    def upload_graph(self, upload_path: str) -> typing.Tuple[bool, str]:
+        """Upload graph and snapshots to robot.
```

Comment:
Same here, could be more descriptive since this is meant to be used externally.

Reply:
Updated the docstring.